internal/vote/run.go

```diff
@@ -305,18 +303,18 @@ func buildBackend(ctx context.Context, env map[string]string, name string) (Back
 			env["VOTE_DATABASE_PORT"],
 			env["VOTE_DATABASE_NAME"],
 		)
-		p, err := postgres.New(ctx, addr)
+		p, err := postgres.New(ctx, addr, env["VOTE_DATABASE_NO_POOL"] != "false")
```

Comment:
This causes the opposite behavior of what the naming infers and also seems unnecessarily confusing.
if `VOTE_DATABASE_NO_POOL=false`, the expression will yield `false` thus disabling the pool.

If go syntax is following other languages standards, I'm pretty sure you meant:
`p, err := postgres.New(ctx, addr, !env["VOTE_DATABASE_NO_POOL"])`

on second thought there's probably some typing needed, something like:
`p, err := postgres.New(ctx, addr, ! bool(env["VOTE_DATABASE_NO_POOL"]))`

might be discarded anyway, see 'Conversation'

Reply:
You are correct. But I also think that this will be discarded.